build.gradle

```diff
@@ -24,6 +29,9 @@ subprojects { Project subproject ->
     }
 
     dependencies {
+        // TODO: Remove when RxJava2 -> Reactor is merged in core
+        api 'io.micronaut.reactor:micronaut-reactor'
```

Comment:
I don'think we should expose micronaut-reactor as api dependency.

Reply:
I've changed to use `reactor-core` not `micronaut-reactor` as an implementation dependency